### PR TITLE
added psm tvl and fees and mod minted

### PR DESCRIPTION
**Pull request by LawsonGraham**

### CODE REVIEW

**Comment by 0xbe1** [comment]
@LawsonGraham can you rebase in order to avoid some format change? also make sure run `yarn prettier` after code change

File: projects/mod/sentio.yaml [hunk 1]
```diff
@@ -1 +1,4 @@
 project: thala-labs/mod
+contracts:
+  - chain: aptos_mainnet
+    address: "0x6f986d146e4a90b828d8c12c14b6f4e003fdff11a8eecceceb63744363eaac01"
```
Comment (0xbe1) [inline projects/mod/sentio.yaml:4]: do we need this? wonder what does it do

File: projects/mod/src/processor.ts [hunk 2]
```diff
@@ -168,13 +199,20 @@ AptosResourcesProcessor.bind({
...
     for (const vaultSum of vaultSums) {
       const coinType = vaultSum.type_arguments[0];
-      const collateral = vaultSum.data_decoded.total_collateral.scaleDown(
+      const psmSum = psmSums.find((psmSum) => psmSum.type_arguments[0] === coinType);
+      const vaultCollateral = vaultSum.data_decoded.total_collateral.scaleDown(
         getCoinInfo(coinType).decimals
       );
+      const psmCollateral = psmSum ? psmSum.data_decoded.coin.value.scaleDown(
+        getCoinInfo(coinType).decimals
+      ) : 0;
```
Comment (0xbe1) [inline projects/mod/src/processor.ts:213]: can we plus then scaleDown?

**Review by 0xbe1 — APPROVED** [review]
lgtm!

File: projects/mod/sentio.yaml (hunk 1 above)
Comment (LawsonGraham) [reply projects/mod/sentio.yaml:4]: idk - it was auto generated when I re-fetched the contract modules

**Pull request merged**